firebase-firestore/src/test/java/com/google/firebase/firestore/remote/BloomFilterTest.java

```diff
@@ -41,13 +41,13 @@ public class BloomFilterTest {
       "src/test/resources/bloom_filter_golden_test_data/";
 
   @Test
-  public void instantiateEmptyBloomFilter() {
+  public void instantiateEmptyBloomFilter() throws BloomFilterException {
```

Comment:
Remove the `throws BloomFilterException` clauses from all methods everywhere, since `BloomFilterException` will be changed to extend `RuntimeException` rather than `Exception`.